### PR TITLE
Completely migrated to Golang 1.15 and Github Actions (fixes #360)

### LINKED ISSUE #360: Migrate to Golang 1.15 and Github Actions
As described.

### PR DESCRIPTION
**Pull request by vania-pooh**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: relea
```
Comment (lanwen) [inline .github/workflows/release.yml:1]:
```suggestion
name: release
```

File: ci/test.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+yarn --cwd ui install
+yarn --cwd ui build
+yarn --cwd ui test
+
+export GO111MODULE="on"
+go get -u github.com/rakyll/statik
+go generate github.com/aerokube/selenoid-ui
```
Comment (lanwen) [inline ci/test.sh:9]: you removed the ui package where generator is executed, so no need to keep it

Comment (vania-pooh) [reply ci/test.sh:9]: Needed to run statik: https://github.com/aerokube/selenoid-ui/blob/master/main.go#L25 Explicit call was removed.

Comment (lanwen) [reply ci/test.sh:9]:
```suggestion
```

File: go.mod [hunk 3]
```diff
@@ -1,11 +1,13 @@
 module github.com/aerokube/selenoid-ui
 
+go 1.15
+
 require (
 	github.com/aandryashin/matchers v0.0.0-20161126170413-435295ea180e
-	github.com/aerokube/util v0.0.0-20181001032247-3a9b3f70da09
+	github.com/aerokube/util v0.0.0-20200816065139-4ca0222e7686
 	github.com/gorilla/websocket v1.4.0 // indirect
+	github.com/hashicorp/go-version v1.2.1 // indirect
```
Comment (lanwen) [inline go.mod:9]: is that used in util?

File: ci/test.sh (hunk 2 above)
Comment (vania-pooh) [reply ci/test.sh:9]: @lanwen removing this line makes no sense.

File: go.mod [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
 	github.com/koding/websocketproxy v0.0.0-20180716164433-0fa3f994f6e7
+	github.com/mitchellh/gox v1.0.1 // indirect
```
Comment (lanwen) [inline go.mod:11]: I think it shouldn't be part of the go.mod, as its just a tool for ci

File: go.sum [hunk 5]
```diff
@@ -1,12 +1,63 @@
...
+github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/rakyll/statik v0.1.7 h1:OF3QCZUuyPxuGEP7B4ypUa7sB/iHtqOTDYZXGM8KOdQ=
 github.com/rakyll/statik v0.1.7/go.mod h1:AlZONWzMtEnMs7W4e/1LURLiI49pIMmp6V9Unghqrcc=
+github.com/sirupsen/logrus v1.4.2/go.mod h1:tLMulIdttU9McNUspp0xgXVQah82FyeX6MwdIuYE2rE=
+github.com/stretchr/objx v0.1.1/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
+github.com/stretchr/testify v1.2.2/go.mod h1:a8OnRcib4nhh0OaRAV+Yts87kKdq0PP7pXfy6kDkUVs=
+golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2/go.mod h1:djNgcEr1/C05ACkg1iLfiJU5Ep61QUkGW8qpdssI0+w=
+golang.org/x/lint v0.0.0-20190313153728-d0100b6bd8b3/go.mod h1:6SW0HCj/g11FgYtHlgUYUwCkIfeOF89ocIRzGO/8vkc=
+golang.org/x/net v0.0.0-20190311183353-d8887717615a/go.mod h1:t9HGtf8HONx5eT2rtn7q6eTqICYqUVnKs3thJo3Qplg=
+golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be/go.mod h1:N/0e6XlmueqKjAGxoOufVs8QHGRruUQn6yWY3a++T0U=
```
Comment (lanwen) [inline go.sum:50]: where all this stuff comes from?

File: go.mod (hunk 4 above)
Comment (vania-pooh) [reply go.mod:11]: Let's just revert updating util.

Comment (vania-pooh) [reply go.mod]: Removed this stuff.

File: go.sum (hunk 5 above)
Comment (vania-pooh) [reply go.sum]: Removed this stuff.

**Comment by vania-pooh** [comment]
@lanwen any more things to fix?

File: ci/test.sh (hunk 2 above)
Comment (lanwen) [reply ci/test.sh:9]: true

File: ui/src/setupProxy.js [hunk 6]
```diff
@@ -7,27 +7,27 @@ const agent = new http.Agent({
...
             target: "http://localhost:8080",
             headers: { Connection: "keep-alive" },
             agent: agent,
         })
     );
-    app.use(proxy("/status", { target: "http://localhost:8080" }));
-    app.use(proxy("/video/", { target: "http://localhost:8080" }));
-    app.use(proxy("/wd/hub/", { target: "http://localhost:8080" }));
-    app.use(proxy("/ws", { target: "http://localhost:8080/", ws: true }));
+    app.use(createProxyMiddleware("/status", { target: "http://localhost:8080" }));
```
Comment (lanwen) [inline ui/src/setupProxy.js:18]: @vania-pooh rebase artifact, shouldn't be in this PR

**Pull request merged**